### ISSUE TITLE
Kirkstone fail to boot imx8mp-hummingboard

### ISSUE THREAD

**Issue by olekslemberg** [SolidRun/meta-solidrun-arm-imx8#10]
Tried to build core-image-minimal at the kirkstone-imx8m branch, and everything was built OK, but after flashing, the board does nothing. No HDMI/miniHDMI output, no ethernet LED blinking, nothing at mikroBUS UART happens, only one pulse in the TX line when the board reboots.

I tried to build both core-image-minimal as well as the imx-image-full.

Also, I build zeus image and tried the prebuilt hardknott minimal image, both are working well

Thanks

UPD: I realized that UART-1 is connected to the micro USB on dev board and it looks like the u-boot performs well, and the booting process is stuck on starting the kernel...

See the boot logs below
```
U-Boot SPL 2022.04-lf_v2022.04+g8640530824 (Jun 07 2022 - 05:16:02 +0000)
Training for 3GByte Micron
DDRINFO: start DRAM init
DDRINFO: DRAM rate 3000MTS
DDRINFO:ddrphy calibration done
DDRINFO: ddrmix config done
SEC0:  RNG instantiated
Normal Boot
Trying to boot from BOOTROM
image offset 0x8000, pagesize 0x200, ivt offset 0x0
NOTICE:  BL31: v2.6(release):lf-5.15.32-2.0.0-0-gc6a19b1a3
NOTICE:  BL31: Built : 06:37:22, Jun  7 2022


U-Boot 2022.04-lf_v2022.04+g8640530824 (Jun 07 2022 - 05:16:02 +0000)

CPU:   i.MX8MP[8] rev1.1 1800 MHz (running at 1200 MHz)
CPU:   Commercial temperature grade (0C to 95C) at 27C
Reset cause: POR
Model: NXP i.MX8MPlus LPDDR4 SolidRun board
DRAM:  3 GiB
Core:  80 devices, 26 uclasses, devicetree: separate
MMC:   FSL_SDHC: 1, FSL_SDHC: 2
Loading Environment from MMC... *** Warning - bad CRC, using default environment

[*]-Video Link 0adv7535_mipi2hdmi adv7535@39: Can't find cec device id=0x3c
fail to probe panel device adv7535@39
fail to get display timings
probe video device failed, ret -19

        [0] lcd-controller@32e80000, video
        [1] mipi_dsi@32e60000, video_bridge
        [2] adv7535@39, panel
adv7535_mipi2hdmi adv7535@39: Can't find cec device id=0x3c
fail to probe panel device adv7535@39
fail to get display timings
probe video device failed, ret -19
In:    serial
Out:   serial
Err:   serial
SEC0:  RNG instantiated

 BuildInfo:
  - ATF c6a19b1

switch to partitions #0, OK
mmc1 is current device
flash target is MMC:1
Net:
Warning: ethernet@30bf0000 (eth1) using random MAC address - ea:9d:1b:0c:94:8d
eth1: ethernet@30bf0000 [PRIME]
Fastboot: Normal
Normal Boot
Hit any key to stop autoboot:  0
switch to partitions #0, OK
mmc1 is current device
Scanning mmc 1:1...
Found /extlinux/extlinux.conf
Retrieving file: /extlinux/extlinux.conf
1:      NXP i.MX Release Distro
Retrieving file: /extlinux/../Image
append: root=/dev/mmcblk1p2 rootwait rw quiet console=ttymxc1,115200  rootwait r                                                                                                                                                             w console=ttymxc1,115200,115200
Retrieving file: /extlinux/../imx8mp-hummingboard-pulse.dtb
## Flattened Device Tree blob at 43000000
   Booting using the fdt blob at 0x43000000
   Using Device Tree in place at 0000000043000000, end 00000000430126f5
adv7535_mipi2hdmi adv7535@39: Can't find cec device id=0x3c
fail to probe panel device adv7535@39
fail to get display timings
probe video device failed, ret -19

Starting kernel ...   
```

**Comment by dhdhxji** [comment]
+1, I faced the exact same issue

**Comment by jnettlet** [comment]
This was fixed with commit a0bc939c4e422042e381dbffda2cb3d9e7324db3, however the kirkstone BSP has now been updated to the latest NXP release, along with most recent kirkstone and oe meta-layers.
